app/views/shared/authentication/_forem_creator_signup.html.erb

```diff
@@ -1,3 +1,94 @@
+              <ul>
+                <% resource.errors.full_messages.each do |message| %>
+                  <li><%= message %></li>
+                <% end %>
+              </ul>
+            </div>
+          </div>
+        <% end %>
+
+        <div class="align-center">
+          <p class="pb-4 fs-3xl fw-bold">Let's start your Forem journey!</p>
+          <p class="color-base-70 fs-xl">Create your admin account first.</p>
+          <p class="color-base-70 fs-xl">Then we'll walk you through your Forem setup.</p>
+        </div>
+
+        <div class="crayons-field mt-6">
+          <%= f.label :name, "Name", for: "name", class: "crayons-field__label" %>
+          <%= f.text_field :name, id: "name", placeholder: "John Doe, john_doe, john, ...", class: "crayons-textfield js-creator-signup-name", required: true %>
+
+          <div class="js-creator-signup-username-hint-row hidden">
+            <div class="fs-xs color-base-60 flex">
+              <span>Your username will be&nbsp;</span>
+              <span class="bold color-base-80 js-creator-signup-username-hint"></span>
+              <button class="cursor-pointer border-none bg-transparent p-0 js-creator-edit-username" role="button">
+                <%= inline_svg_tag("pencil.svg", class: "cursor-pointer", aria: true, title: "Pencil icon") %>
+              </button>
+            </div>
+            </div>
+        </div>
+
+        <div class="crayons-field mt-6 <%= "hidden" unless any_errors %> js-creator-signup-username-row">
+          <%= f.label :username, "Username", for: "username", class: "crayons-field__label" %>
+          <%= f.text_field :username, pattern: "[A-Za-z0-9_]+", maxlength: "30", title: "A username can only contain letters, numbers and underscores. The maximum username length can be 30 characters.", id: "username", class: "js-creator-signup-username crayons-textfield" %>
+        </div>
+
+        <div class="crayons-field mt-6">
+          <%= f.label :email, "Email", for: "email", class: "crayons-field__label" %>
+          <%= f.email_field :email, placeholder: "john@example.com", autocomplete: "email", id: "email", class: "crayons-textfield", required: true %>
+        </div>
+
+        <div class="crayons-field mt-6 relative">
+          <%= f.label :password, class: "crayons-field__label", aria: { label: "Password" } do %>
+            Password
+            <span class="crayons-field__required crayons-tooltip"></span>
+          <% end %>
+          <span class="js-creator-password-visibility">
+            <%= inline_svg_tag("eye.svg", class: "absolute bottom-2 right-2 cursor-pointer js-eye", aria: true, title: "Eye icon") %>
+            <%= inline_svg_tag("eye-off.svg", class: "absolute bottom-2 right-2 cursor-pointer hidden js-eye-off", aria: true, title: "Eye Off icon") %>
+          </span>
+          <%= f.password_field :password, minlength: "8", class: "crayons-textfield js-password", required: true %>
```

Comment:
This is more a UX/Design thing but just a general thought is that it tends to be better to prevent users from making errors, rather than correcting them afterwards. Maybe in a future iteration we could consider helper text below inputs explaining the validation e.g. "Minimum 8 characters"

Reply:
@aitchiss I agree on this, I'll add it something that we iterate on in #pod-admin-experience

Reply:
@ludwiczakpawel any chance you can add some subtext under the original design to outline these validations and I can possibly add it sooner rather than later?

Reply:
let me dedust my figma, and i'll add it 👍

Reply:
Thanks @ludwiczakpawel!

Reply:
@Ridhwana I've added that little hint text [here](https://www.figma.com/file/VRmjMjAbRWmNzyzLZWS77q/Admin-onboarding?node-id=138%3A1128).